streaming-binlog/src/main/java/org/openmrs/analytics/DbeziumArgs.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package org.openmrs.analytics;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+@Parameters(separators = "=")
+public class DbeziumArgs {
```

Comment:
If this class is intended to be used only in `DebeziumListener` consider moving it there as an inner class.

Alternatively, it seems that some of the arguments are shared with other cases (like atom-feed based streaming), e.g., OpenMRS username/password, so maybe you can create a base class with the shared arguments and a subclass `DebeziumArgs` here where you only include Debezium specific args.

Reply:
fixed